<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/bin/conda-lint", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/run.py", line 92, in main
    result = linter.lint(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 605, in lint
    msgs = self.lint_one(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/lint/__init__.py", line 653, in lint_one
    recipe = _recipe.Recipe.from_file(
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/recipe.py", line 268, in from_file
    raise exc
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/recipe.py", line 259, in from_file
    recipe.load_from_string(text.read(), selector_dict)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/recipe.py", line 207, in load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/anaconda_linter/recipe.py", line 403, in render
    self.meta = yaml.load(yaml_text)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/constructor.py", line 119, in get_single_data
    node = self.composer.get_single_node()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 76, in get_single_node
    document = self.compose_document()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 99, in compose_document
    node = self.compose_node(None, None)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 143, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 223, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 141, in compose_node
    node = self.compose_sequence_node(anchor)
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/composer.py", line 184, in compose_sequence_node
    while not self.parser.check_event(SequenceEndEvent):
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/parser.py", line 146, in check_event
    self.current_event = self.state()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/parser.py", line 526, in parse_block_sequence_first_entry
    return self.parse_block_sequence_entry()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/parser.py", line 533, in parse_block_sequence_entry
    if not self.scanner.check_token(BlockEntryToken, BlockEndToken):
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/scanner.py", line 1794, in check_token
    while self.need_more_tokens():
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/scanner.py", line 211, in need_more_tokens
    self.stale_possible_simple_keys()
  File "/opt/conda/envs/linter/lib/python3.10/site-packages/ruamel/yaml/scanner.py", line 360, in stale_possible_simple_keys
    raise ScannerError(
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 36, column 3:
      pytest disabled pending trustme  ...
      ^ (line: 36)
could not find expected ':'
  in "<unicode string>", line 37, column 3:
      source_files:
      ^ (line: 37)

ERROR conda.cli.main_run:execute(49): `conda run conda-lint /tmp/abs_d6quofi8v1/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -41,7 +41,6 @@ test:
   requires:
     - pip
     - pytest >=6.1
-    - pytest-httpserver >=1.0.4
     - flask >=2.1.2
     - requests-wsgi-adapter >=0.4.1
     - trustme >=0.9.0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -36,7 +36,8 @@ test:
   source_files:
     - tests
   commands:
-    - pytest tests
+  # Skipping test_session_with_selfsigned_ca because pytest-httpserver is not on defaults
+    - pytest tests -k "not test_session_with_selfsigned_ca"
     - pip check
   requires:
     - pip

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -33,7 +33,6 @@ requirements:
 test:
   imports:
     - unearth
-  pytest disabled pending trustme 0.9.0
   source_files:
     - tests
   commands:

